datamodel_code_generator/parser/jsonschema.py

```diff
@@ -183,10 +184,21 @@ def get_data_type(self, obj: JsonSchemaObject) -> List[DataType]:
         if isinstance(obj.type, list):
             types: List[str] = [t for t in obj.type if t != 'null']
```

Comment:
I'm curious if the string `'null'` needs to be stripped here for some weird inputs (e.g. `'  nUlL     '`)

Reply:
I guess I don't need stripping these inputs.
`null` is defined in the rule book.